Cargo.toml

```diff
@@ -20,10 +20,11 @@ pomelo = { version = "0.1.4", optional = true }
 thiserror = "1.0.21"
 serde = { version = "1.0", features = ["derive"], optional = true }
 petgraph = { version ="0.5", optional = true }
+pp-rs = { git = "https://github.com/Kangz/glslpp-rs", rev = "4f2f72a", optional = true }
```

Comment:
what happened here? maybe we are missing a proper rebase?

Reply:
Hmm, I may have screwed up the rebase. Let me try that again,

Reply:
This is in master already.